tests/expr/cat/get_categories_test.py

```diff
@@ -6,24 +6,27 @@
 import pytest
 
 import narwhals.stable.v1 as nw
+from narwhals.utils import parse_version
 from tests.utils import compare_dicts
 
 data = {"a": ["one", "two", "two"]}
 
 
 def test_get_categories(request: Any, constructor: Any) -> None:
-    if "pyarrow_table" in str(constructor):
+    if "pyarrow_table" in str(constructor) and parse_version(
+        pa.__version__
+    ) <= parse_version("11.0.0"):
```

Comment:
I am not sure which is the min version for which it doesn't break... we test 11 specifically, and the rest auto-installs